packages/vue-renderer/src/renderer.js

```diff
@@ -348,39 +378,57 @@ export default class VueRenderer {
     }
   }
 
-  static parseTemplate(templateStr) {
-    return template(templateStr, {
-      interpolate: /{{([\s\S]+?)}}/g
-    })
-  }
-
-  static get resourceMap() {
-    return [
-      {
-        key: 'clientManifest',
-        fileName: 'vue-ssr-client-manifest.json',
-        transform: JSON.parse
+  get resourceMap() {
+    return {
+      clientManifest: {
+        fileName: 'client.manifest.json',
+        transform: src => JSON.parse(src)
       },
-      {
-        key: 'modernManifest',
-        fileName: 'vue-ssr-modern-manifest.json',
-        transform: JSON.parse
+      modernManifest: {
+        fileName: 'modern.manifest.json',
+        transform: src => JSON.parse(src)
       },
-      {
-        key: 'serverBundle',
-        fileName: 'server-bundle.json',
-        transform: JSON.parse
+      serverManifest: {
+        fileName: 'server.manifest.json',
+        // BundleRenderer needs resolved contents
+        transform: (src, { readResource, oldValue = { files: {}, maps: {} } }) => {
```

Comment:
Can we use `this.context.resources. serverManifest ` instead of oldValue ?

Reply:
Exactly. I just wanted to make transforms contextless :)